### DISCUSSION TITLE
Can I generate the markdown files for docusaurus without executing `build` or `start`?

### DISCUSSION THREAD

**Discussion by lengors** [tgreyuk/typedoc-plugin-markdown#490] [Help] [answered]
I want to preface this by clarifying what I'm trying to do and how I'm currently approaching it.

So, for what I'm trying to do is that I'm trying to use the docusaurus typedoc plugin to generate the documentation for my typescript project so that then I can use docusaurus builtin support for versioning.
The way I'm doing it right now is by calling the `build` script (so typedoc generates the markdown files), tag a version (`npm run docusaurus docs:version 1.0.0`) and re-run the `build` command.

So, as the title says, with the docusaurus plugin, can I generate the markdown files for docusaurus without executing the `build` or `start` scripts?

**Comment by tgreyuk** [comment, marked as answer]
There is an additional custom cli command for this `docusaurus generate-typedoc`.

Apologies it doesn't appear to be in the docs but i'll add it.

**Reply by lengors** [reply]
Thank you 🙂
